Update README.md

1Mill opened the pull request:
Improved the readability of the README file by reformatting various sections. 

Reformatted the installation steps to be clearer about what the user must do, and put where, in order to make FullCalendar display on the screen.

Merged the Usage section with the introductory paragraph.

Added Using FullCalendar section to link to helpful resources.

gr8bit commented:
What about the original print CSS instructions with the hint to issue #11 ? Do you think it isn't helpful or isn't it valid anymore? Also, I think the section about the versioning ("X.Y.Z") can be misunderstood as the gem's version is (and always should be) in sync with FullCalendar's version, except for the 4th digit.
I like the rest! +1 for merging - what @bokmann, @davidwessman, your thoughts?

davidwessman commented on README.md:
````diff
@@ -1,82 +1,76 @@
-Add this line to your application's Gemfile:
-
+1. Add to `gemfile`
+    ```
````
```ruby 😄

davidwessman commented:
I like the new formats.
I think there should be a comment about the Gem-versioning which do follow the version of Fullcalendar.

I am not using the `fullcalendar.print.css` so I am not quite sure.
But since it has been an issue before I think it should be mentioned.

1Mill commented:
Changed the wording of the specified gem version install to represent the version of the gem the user is getting, but more importantly to highlight the version of FullCalendar they are installing. 

I too don't use fullcalendar.print, so I am not entirely sure how to handle it lol. But nevertheless I updated the installation steps to highlight issue#11 and included the "ideal solution" from the already given master readme document.

gr8bit commented:
Thank you, I think it's very good now 👍
